### PR TITLE
Promisify and support Node.js >=4

### PR DESCRIPTION
**Pull request by kevva**
Couldn't use `get-stream` here since we need to return an array. Also bump AVA and adds XO.

### CODE REVIEW

**Comment by SamVerschueren** [comment]
Would it be possible to add a test for an error? Seems like a good extra.

**Comment by kevva** [comment]
> Would it be possible to add a test for an error? Seems like a good extra.

Good idea.

**Comment by sindresorhus** [comment]
Awesome! Thank you @kevva :)

File: readme.md [hunk 1]
```diff
@@ -41,14 +43,6 @@ Type: `object`
...
-#### callback(error, data)
-
-##### data
-
-Type: `array`
-
-Parsed CSV.
-
 
 ## License
```
Comment (sindresorhus) [inline readme.md:48]: Btw, I also change my URL to `https` now ⬇️

Comment (kevva) [inline readme.md:48]: Duly noted :haircut:

**Pull request merged**